elements/pfe-content-set/package.json

```diff
@@ -28,8 +28,5 @@
     "@patternfly/pfe-tabs": "^1.0.0-prerelease.9",
     "@patternfly/pfelement": "^1.0.0-prerelease.9"
   },
-  "devDependencies": {
-    "@patternfly/pfe-sass": "^1.0.0-prerelease.9"
```

Comment:
Why remove pfe-sass as a dev dependency?

Reply:
@castastrophe We're not using it. We're just using

```
:host {
  display: block;
}

:host([hidden]) {
  display: none;
}
```

Reply:
Oh of course.  This is a combo. #duh